typhos/cli.py

```diff
@@ -79,24 +79,28 @@ def typhos_cli_setup(args):
             app.setStyleSheet(handle.read())
 
 
+def _create_happi_client(cfg):
+    """Create a happi client based on configuration ``cfg``."""
+    import happi
+    import typhos.plugins.happi
+
+    if typhos.plugins.happi.HappiClientState.client:
+        logger.debug("Using happi Client already registered with Typhos")
+        return typhos.plugins.happi.HappiClientState.client
+
+    logger.debug("Creating new happi Client from configuration")
+    return happi.Client.from_config(cfg=cfg)
+
+
 def create_suite(devices, cfg=None, fake_devices=False):
     """Create a TyphosSuite from a list of device names"""
     logger.debug("Accessing Happi Client ...")
-    happi_enabled = False
+
     try:
-        import happi
-        import typhos.plugins.happi
-        happi_enabled = True
-    except (ImportError, ModuleNotFoundError):
-        logger.exception("Unable to import happi to load devices!")
-
-    if happi_enabled:
-        if typhos.plugins.happi.HappiClientState.client:
-            logger.debug("Using happi Client already registered with Typhos")
-            client = typhos.plugins.happi.HappiClientState.client
-        else:
-            logger.debug("Creating new happi Client from configuration")
-            client = happi.Client.from_config(cfg=cfg)
+        happi_client = _create_happi_client(cfg)
+    except Exception:
+        logger.debug("Unable to create a happi client.")
```

Comment:
One thought: this has the potential to hide obscure misconfiguration bugs, and drops any known exceptions and their accompanying error messages from `happi.Client.from_config`. The output is a bit messy but I suggest including the `exc_info` in the debug statement here.
```suggestion
        logger.debug("Unable to create a happi client.", exc_info=True)
```